packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -129,8 +130,15 @@ export const render = async (req, res, next) => {
 
     AppConfig.restore(res.locals)
 
+    // Use locals to thread the application extensions through the rendering pipeline.
+    res.locals.appExtensions = extensions
```

Comment:
In my previous change, i store the server extensions in the app object, see https://github.com/SalesforceCommerceCloud/pwa-kit/blob/feature/extensibility-v2/packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js#L663
When I wrote it, i imagine extensions are internals, and we don't need to expose it to the userland, but I think my assumption may need to change.

Let's pick a place, `app.__extensions` or `res.locals.appExtensions`

Reply:
If you look at the code, I placed the extensions inside `locals` because I didn't want to dirty up the signature for getting the routes in `getRoutes`. I wasn't thinking about how to make them internal or public.. really only wanted to use them in the getRoutes function without having to change the signature. Does that change your comment at all?